dist/about-thjis-directory

```diff
@@ -0,0 +1,5 @@
+This dist/ directory is here to maintain backwards compatibility with sites that use the pre-2.0 version of Water.css.
```

Comment:
Hmm, I'd call this README so GitHub displays it automatically if someone clicks on `dist/` in the file tree.
Or do you think having a semantic file name (typo btw :D) is more important?

Reply:
Oh, forgot GitHub does that. I shall rename.